N2A/src/gov/sandia/n2a/ui/settings/SettingsRepo.java

```diff
@@ -1393,8 +1385,8 @@ public Delta (GitWrapper git)
         public MDir getDir ()
         {
             String repoName = git.gitDir.getParent ().getFileName ().toString ();
-            if (name.startsWith ("models")) return getModels     (repoName);
-            else                            return getReferences (repoName);
+            String[] pieces = repoName.split ("/");
```

Comment:
repoName is the simple name of the repo directory. There's no need to split it. Instead, you want
    return getExisting(name, repoName);